blocks/components/editable/index.js

```diff
@@ -1,3 +1,98 @@
-export default function Editable( { value } ) {
-	return wp.element.createElement( 'p', null, value );
+const el = wp.element.createElement;
+let editorInstance = 1;
+
+export default class Editable extends wp.element.Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onInit = this.onInit.bind( this );
+		this.onSetup = this.onSetup.bind( this );
+		this.onChange = this.onChange.bind( this );
+		this.onFocusIn = this.onFocusIn.bind( this );
+		this.onFocusOut = this.onFocusOut.bind( this );
+		this.id = `tinymce-instance-${ editorInstance }`;
+		editorInstance++;
```

Comment:
Doesn't make a whole lot of difference, but we could assign the incremented value as a static property of the class:

```js
export default class Editable extends wp.element.Component {
	static instance = 1;

	constructor() {
		// ...
		this.id = `tinymce-instance-${ this.constructor.instance }`;
		this.constructor.instance++;

		// or: (increment slightly less obvious)
		this.id = `tinymce-instance-${ this.constructor.instance++ }`;
	}
```